lib/db/mem.js

```diff
@@ -1362,6 +1363,66 @@ module.exports = function (log, error) {
       })
   }
 
+  Memory.prototype.createRecoveryKey = function (uid, data) {
+    uid = uid.toString('hex')
+    const recoveryKeyId = data.recoveryKeyId.toString('hex')
+    return getAccountByUid(uid)
+      .then(() => {
+        if (! recoveryKeys[uid]) {
+          recoveryKeys[uid] = {}
+        }
+
+        const storedKeys = recoveryKeys[uid]
+
+        if (Object.keys(storedKeys).length > 1) {
```

Comment:
Should this check for `.length > 0` rather than `.length > 1`?  Otherwise we will allow the store to contain up to two keys before erroring out.